authors/settings.py

```diff
@@ -11,6 +11,7 @@
 """
 
 import os
+import django_heroku
```

Comment:
Avoid use of preset libraries, for your own learning well being.

Reply:
Removed django_heroku package and now reconfigured for manual deploy.